modules/dxp/apps/analytics/analytics-reports-api/bnd.bnd

```diff
@@ -1,4 +1,6 @@
 Bundle-Name: Liferay Analytics Reports API
 Bundle-SymbolicName: com.liferay.analytics.reports.api
 Bundle-Version: 1.0.5
-Export-Package: com.liferay.analytics.reports.info.item
\ No newline at end of file
+Export-Package:\
+	com.liferay.analytics.reports.info.action,\
```

Comment:
What do you think about calling the package info.item.action?

Reply:
I was tempted to do it , I used this one to match the content-dashboard one. But, now once this API is Analytics Reports related I prefer you proposal